sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -1,37 +1,37 @@
 <template>
-  <student-page-container :full-width="true">
+  <student-page-container>
     <template #header>
       <header-navigator
         title="Applications"
         subTitle="Financial Aid Application"
       >
+        <template #buttons>
+          <v-row class="m-0 p-0 float-right">
```

Comment:
i don't think we need `float-right` the slot already has it, i believe

Reply:
I believe that even the `<v-row>` can be part of the component and wrap the `<slot>`. Not part of this PR, only if you are willing to 😉

Reply:
Yeah, ok. Removed the v-row. We actually don't need it. Thanks!